app/crds.py

```diff
@@ -128,6 +141,7 @@ class ResourceSpec(BaseModel):
     is_visible: bool = True
     is_browser_shortcut_enabled: bool = True
     protocols: ResourceProtocols = Field(default_factory=ResourceProtocols)
+    tags: list[ResourceTag] = Field(default_factory=list, exclude=True)
```

Comment:
Technically, `tags` is not part of [`TwignateResource.spec`](https://github.com/Twingate/kubernetes-operator/wiki/API-Reference#twingateresourcespec), so we should exclude this field by default during model export.

Reply:
I updated the implementation, let me know what you think?